### ISSUE TITLE
[BOUNDARY ERRATA] NLD boundaries including waters

### ISSUE THREAD

**Issue by BrentWHR** [wmgeolab/geoBoundaries#2654]
The new 2022 boundaries for NLD introduced after #2444 in #2495  #2496 and #2497 include water (large lakes and seas). The lakes and sees within the boundaries can be seen in attached screenshot with NLD AMD1.

Only later did I realise geoBoundaries offers terrestrial boundaries. I'm sorry for this oversight. 
As far as I can tell, the most reliable maps of Dutch terrestrial administrative boundaries are produced by the national statistics agency (CBS) and can be accessed via WFS here: https://www.pdok.nl/geo-services/-/article/cbs-gebiedsindelingen#7cf43241c34101005f838257ce110a2d
The features you will want are 'gemeente_niet_gegeneraliseerd' for ADM2 and 'provincie_gegeneraliseerd' for AMD1.
This source does not include a boundary on ADM0. Perhaps reverting #2497 would suffice. As far as I know there haven't been any national border adjustment in recent times.

![afbeelding](https://user-images.githubusercontent.com/72964508/209143522-a19a29d5-1725-495d-b60a-6bf3ca39afaf.png)

Please let me know if I can be of further assistance.

**Comment by DanRunfola** [comment]
Thanks for raising this - I'm going to leave it be for now, and we'll just recognize it as an issue until this is resolved.  I'd like to contrast what we had prior, as I'd like to do at least a little research here to make sure we have something accurate.  The other solution could also be to clip to water borders from another source.

**Comment by DanRunfola** [comment]
@all-contributors add @BrentWHR to ideas

**Comment by DanRunfola** [comment]
I am working on a global post to aggregate some of these issues, so am closing this one out.  Expect a new issue that covers this topic by EOB today.
